libs/credentials/src/validators/easAttestations/index.ts

```diff
@@ -51,67 +52,62 @@ const validator: Validator = {
      * @returns True if the user meets the criteria.
      */
     async validate(criteria: Criteria, context: Context) {
-        if ("queryGraph" in context) {
-            const getAttestations = (context as EASContext).queryGraph
-
+        if ("network" in context) {
             const {
-                recipient,
+                minAttestations,
                 attester,
                 schemaId,
                 revocable,
                 revoked,
                 isOffchain
             } = criteria
 
-            const attestations = await getAttestations(`
-                query {
-                    attestations {
-                        recipient
-                        attester
-                        revocable
-                        revoked
-                        schemaId
-                        isOffchain
+            const query = `query {
```

Comment:
Instead of querying inside the query, what about creating the where conditions before the query so that there are not empty spaces in the result query when some properties are undefined or null. Something like:

```ts
let whereConditions = `
  recipient: { equals: "${context.address}" }
`;

if (attester !== undefined && attester !== null) {
  whereConditions += `attester: { equals: "${attester}" },`;
}

if (schemaId !== undefined && schemaId !== null) {
  whereConditions += `schemaId: { equals: "${schemaId}" },`;
}

if (revocable !== undefined && revocable !== null) {
  whereConditions += `revocable: { equals: ${revocable} },`;
}

if (revoked !== undefined && revoked !== null) {
  whereConditions += `revoked: { equals: ${revoked} },`;
}

if (isOffchain !== undefined && isOffchain !== null) {
  whereConditions += `isOffchain: { equals: ${isOffchain} },`;
}

const query = `query {
  attestations(where: { ${whereConditions} }) {
    recipient
    attester
    schemaId
    revocable
    revoked
    isOffchain
  }
}`;

```